src/_pytest/python.py

```diff
@@ -1586,6 +1587,7 @@ def __init__(
         # to a readonly property that returns FunctionDefinition.name.
 
         self.keywords.update(self.obj.__dict__)
+        self.own_markers.extend(get_all_mro_marks(self.cls, self.module.__name__))
```

Comment:
mro markers should be added to class instances, not functions

Reply:
Done